volatility3/framework/plugins/windows/dlllist.py

```diff
@@ -204,7 +147,7 @@ def _generator(self, procs):
 
                 file_output = "Disabled"
                 if self.config["dump"]:
-                    file_handle = self.dump_pe(
+                    file_handle = pedump.PEDump.dump_ldr_entry(
```

Comment:
This isn't really a file_handle anymore?  Could we get the variable name updated to something more appropriate please?

Reply:
fixed